new-york-city.md

```diff
@@ -19,6 +19,8 @@ Lunch options near the office:
 
 * [All'onda](http://allondanyc.com) – Union Square, Mexican
 * Bryant Market - 38th and 6th Ave, Bagels
+* [Bryant Park Grill](http://www.bryantpark.org/things-to-do/bryant_park_grill.html) –
```

Comment:
the menu and more info is available here: http://arkrestaurants.com/bryant_park/